driverinterface.sh

```diff
@@ -1,13 +1,8 @@
 #!/bin/sh 
-cd /home/ucrsolarcar/UCRSolarCar2023/Driver-Interface-2022
 
-res=$(git pull origin main | grep Already)
-if [[ ${res} =~ 'Already' ]]
-then
-  echo 'No updates have been made :)'
-else
-  qmake DriverInterface.pro
-  make
-fi
-
-./DriverInterface
+for FILE in *; do
```

Comment:
Have you tested this out?

Reply:
I did test this out. All .cpp and .h files get formatted. Did you find an error or see that it is incorrect?